assets/src/block-validation/components/error/get-error-source-title.js

```diff
@@ -86,7 +90,7 @@ export function getErrorSourceTitle( sources = [] ) {
 		output.push( __( 'Core', 'amp' ) );
 	}
 
-	if ( 0 === output.length && 0 < sources.length ) {
+	if ( 0 === output.length && 0 < sources?.length ) {
```

Comment:
If `sources` is `null` then is the resulting effect being `0 < null`? If so, would this be better as:

```suggestion
	if ( 0 === output.length && Array.isArray( sources ) && 0 < sources.length ) {
```

Or otherwise:

```suggestion
	if ( ! output.length && ! sources?.length ) {
```

Reply:
That's a good catch. I like the latter suggestion more.